docker-compose.yml

```diff
@@ -69,21 +71,20 @@ services:
       timeout: 4s
       retries: 100
 
-  jaeger:
-    <<: *autoRestartOnFailure
-    image: jaegertracing/all-in-one:1.62.0
-    ports:
-      - "16686:16686"
-      - "4317:4317"
-      - "4318:4318"
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.116.1
     environment:
-      COLLECTOR_ZIPKIN_HOST_PORT: ":9411"
-    healthcheck:
-      test: ["CMD-SHELL", "/go/bin/all-in-one-linux status"]
-      interval: 3s
-      timeout: 3s
-      retries: 10
-      
+      OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED: "true"
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
+    ports:
+      - 1888:1888 # pprof extension
+      - 8888:8888 # Prometheus metrics exposed by the Collector
+      - 8889:8889 # Prometheus exporter metrics
+      - 13133:13133 # health_check extension
+      - 4317:4317 # OTLP gRPC receiver
+      - 4318:4318 # OTLP http receiver
```

Comment:
Are both supported by mindsdb in terms of exporting?

Reply:
Now it is.